### PR TITLE
Enabling scroll in treeview

### PR DESCRIPTION
**Pull request by hbaptiste**
Enabling scroll in treeview

### CODE REVIEW

**Comment by fkroockmann** [comment]
We need to ask a tag to a jqTree lib

**Comment by mickaelandrieu** [comment]
:+1: we wont merge a contribution that depends on a dev branch! What if finaly the guy close this branch ? we are in mess.

**Comment by hbaptiste** [comment]
Thanks for your insight!

File: bower.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "i18next": "*",
     "nunjucks": "~1.1.0",
     "jquery-ui": "~1.11.2",
-    "jqtree": "~0.22.0",
+    "jqtree": "~1.2.1",
```
Comment (mickaelandrieu) [inline bower.json]: :+1: far better, thanks ! let's hope tests will pass and that the jqtree still works as expected :)

**Comment by fkroockmann** [comment]
:+1:

**Pull request merged**